pom.xml

```diff
@@ -138,6 +138,11 @@
         <ddf-solr>DDF Solr</ddf-solr>
         <ddf-spatial>DDF Spatial</ddf-spatial>
         <ddf-ui>DDF Search UI</ddf-ui>
+
+        <asciidoctor.maven.plugin.version>1.5.2</asciidoctor.maven.plugin.version>
```

Comment:
I was going to say we can probably move this all to the docs module, but looks like it is used in the search-ui as well.